src/Pages/junheekang/Login/Login.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import '../../junheekang/Login/Login.scss';
+
+class Login extends React.Component {
+  goToMain = () => {
+    this.props.history.push('/main-kang');
+  };
+
+  constructor() {
+    super();
+
+    this.state = {
+      userId: '',
+      userPassword: '',
+    };
+  }
+
+  handleIdInput = e => {
+    this.setState({
+      userId: e.target.value,
+    });
+  };
+
+  handlePasswordInput = e => {
+    this.setState({
+      userPassword: e.target.value,
+    });
+  };
+
+  render() {
+    const { userId, userPassword } = this.state;
+    const { handleIdInput, handlePasswordInput } = this;
+    const disabled = userId.includes('@') && userPassword.length >= 5;
```

Comment:
```suggestion
    const validate = userId.includes('@') && userPassword.length >= 5;
```
주니언니 안녕 🤓   맥락상 `true` 값이 유효한 값이지 않을까요?!